scripts/update_packages.py

```diff
@@ -2,6 +2,7 @@
 """Update conda packages on anaconda.org with latest versions for multiple platforms.
```

Comment:
Not for your builds but I'm still finding them useful for testing and debugging. I'm hoping to phase them out over time and will definitely pull them out after more testing.